src/web_app/js/sdputils.js

```diff
@@ -330,9 +330,15 @@ function maybePreferCodec(sdp, type, dir, codec) {
   }
 
   // If the codec is available, set it as the default in m line.
-  var payload = getCodecPayloadType(sdpLines, codec);
-  if (payload) {
-    sdpLines[mLineIndex] = setDefaultCodec(sdpLines[mLineIndex], payload);
+  var payload = null;
+  for (var i = 0; i < sdpLines.length; i++) {
+    var index = findLineInRange(sdpLines, i, -1, 'a=rtpmap', codec);
+    if (index !== null) {
+      payload = getCodecPayloadTypeFromLine(sdpLines[index]);
+      if (payload) {
+        sdpLines[mLineIndex] = setDefaultCodec(sdpLines[mLineIndex], payload);
+      }
```

Comment:
SDP lines from ```i``` to ```index``` was scanned in ```findLineInRange```. Perhaps we can skip them by adding ```i = index;``` here.

Reply:
@jianjunz I originally had exactly that, but removed it thinking it was redundant.  You're right though it would skip the next index - i iterations. I'll update